packages/backend/src/api-impl.ts

```diff
@@ -275,11 +275,11 @@ export class BootcApiImpl implements BootcApi {
   // The API does not allow callbacks through the RPC, so instead
   // we send "notify" messages to the frontend to trigger a refresh
   // this method is internal and meant to be used by the API implementation
-  protected async notify(msg: string, body?: unknown): Promise<void> {
+  protected async notify(msg: string, body: unknown): Promise<void> {
```

Comment:
maybe you should use the following pattern
```suggestion
  protected async notify(msg: string, body: unknown = {}): Promise<void> {
```

Reply:
Excellent point, fixed in new commit.